Source/WebCore/rendering/svg/LegacyRenderSVGShape.cpp

```diff
@@ -174,24 +173,19 @@ void LegacyRenderSVGShape::layout()
     clearNeedsLayout();
 }
 
-Path* LegacyRenderSVGShape::nonScalingStrokePath(const Path* path, const AffineTransform& strokeTransform) const
+Path LegacyRenderSVGShape::nonScalingStrokePath(const Path& path) const
 {
-    static NeverDestroyed<Path> tempPath;
-
-    tempPath.get() = *path;
-    tempPath.get().transform(strokeTransform);
-
-    return &tempPath.get();
+    return nonScalingStrokeTransform().mapPath(path);
```

Comment:
.. however this is simple enough that the function call overhead could be saved by omitting the helper function altogether.